CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing guidelines
+
+- [Introduction](#introduction)
+- [Pull Requests (PRs)](#pull-requests-prs)
+  - [Etiquette for creating PRs](#etiquette-for-creating-prs)
+  - [Checklist for publishing PRs](#checklist-for-publishing-prs)
+- [Setting up a local development
+  environment](#setting-up-a-local-development-environment)
+  - [Code checks with pre-commit](#code-checks-with-pre-commit)
+  - [Unit tests with pytest](#unit-tests-with-pytest)
+
+## Introduction
+
+Thank you for contributing to SHAP. SHAP is an open source collective effort,
+and contributions of all forms are welcome!
+
+You can contribute by:
+
+- Submitting bug reports and features requests on the GitHub issue tracker
+- Contributing fixes and improvements via Pull Requests
+
+## Pull Requests (PRs)
```

Comment:
I'm thinking the PR section should come last in this document (just so it fits the logical flow of things -- new contributors will need to set up their environment before even being concerned with PR etiquette)